README.md

```diff
@@ -0,0 +1,26 @@
+# moviedialogs api
+
+## Setup
+
+* `pipenv install`
+* create `.env` with commands or run manually:
+  * `export DATABASE_URI=<dialect://user:password@host/dbname>`
+  * `export FLASK_APP=api`
+  * `export FLASK_ENV=development` (enable development environment with debug mode)
+* `pipenv shell`
+* `flask db upgrade`
+* `flask seed`
+
+`flask run` to run app on `http://localhost:5000/`
+
+## API Endpoints
+
+|Method | URL | Action |
+|-------|-----|--------|
+|GET    | /moviedb/api/characters/[string:character_id] | Retrieve a character |
```

Comment:
We should remove `/moviedb` from the path imo, the domain name will add that information